util/example/line.json

```diff
@@ -2,6 +2,6 @@
 { "id": 2, "type": "vertex", "label": "range" }
 { "id": 3, "type": "vertex", "label": "range", "tag": { "text": "foo" } }
 { "id": 4, "type": "edge", "label": "contains", "outV": 1, "inV": 2 }
-{ "id": 5, "type": "edge", "label": "contains", "outV": 1, "inV": 3 }
+{ "id": 5, "type": "edge", "label": "contains", "outV": 1, "inVs": [ 3 ] }
 { "id": 6, "type": "vertex", "label": "referenceResult" }
-{ "id": 7, "type": "edge", "label": "item", "property": "references", "outV": 6, "inV": 3 }
\ No newline at end of file
+{ "id": 7, "type": "edge", "label": "item", "property": "references", "outV": 6, "inVs": [3], "document": "1" }
```

Comment:
Ultra nit: use consistent spacing with arrays (line 7 and line 5).

Reply:
Hey, why not. I also cleaned up the formatting for the JSON array file.